src/masonite/billing/drivers/BillingStripeDriver.py

```diff
@@ -1,23 +1,29 @@
-""" The Stripe Billing Driver """
-
-import pendulum
 import stripe
 from stripe.error import InvalidRequestError
 
-from billing.exceptions import PlanNotFound
-
-try:
-    from config import billing
-
-    stripe.api_key = billing.DRIVERS["stripe"]["secret"]
-except ImportError:
-    raise ImportError("Billing configuration found")
+from ..exceptions import PlanNotFound, InvalidDriverConfiguration
 
 
 class BillingStripeDriver:
+    """Stripe billing driver"""
 
     _subscription_args = {}
 
+    def __init__(self, application):
+        self.application = application
+        self.options = {}
+
+    def set_options(self, options):
+        self.options = options
+        api_key = self.options.get("secret")
+        if api_key:
+            stripe.api_key = api_key
+        else:
+            raise InvalidDriverConfiguration(
+                "Stripe API key not found. Please provide 'secret' in config/billing.py."
```

Comment:
Technically the location of the configuration files in Masonite 4 could not be in "config/billing.py". Should probably change this to "in your billing configuration file"